components/NavBar.js

```diff
@@ -244,20 +243,37 @@ const NavBar = () => {
         setOpacity('1');
       }
       lastScroll = scroll;
-    }
-  }
+    };
+  };
 
   if (showDropdown) {
     return (
       <>
         <NavBarContainer darkBg>
-          <NwPlusLogo src="/assets/logos/nwPlus_Logo_2020.svg" alt="nwPlus club logo in white against dark blue background"/>
-          <Cross src="/assets/icons/close_white.svg" alt="dropdown menu icon" onClick={() => setShowDropdown(false)}/>
+          <a href='https://nwplus.io'>
```

Comment:
link to homepage

Reply:
This can be set as `/` and it will go back to the index page for both dev and prod environments!